ipod-shuffle-4g.py

```diff
@@ -423,18 +440,23 @@ def construct(self, tracks):
 
         # Build all the remaining playlists
         playlistcount = 1
+        podcastlistcount = 0
         for i in self.lists:
             playlist = Playlist(self)
             verboseprint("[+] Adding playlist", (i[0] if type(i) == type(()) else i))
             playlist.populate(i)
             construction = playlist.construct(tracks)
             if playlist["number_of_songs"] > 0:
+                if playlist["listtype"] == 3:
+                    podcastlistcount += 1
                 playlistcount += 1
                 chunks += [construction]
             else:
                 print("Error: Playlist does not contain a single track. Skipping playlist.")
 
         self["number_of_playlists"] = playlistcount
+        if podcastlistcount > 0:
```

Comment:
I think the if is not required, just always set the value?

Reply:
Okay I see, we should keep 0xFFFF as a default. Can you maybe add a comment that explains why this if exists?

Reply:
will do